packages/connect-thegraph-voting/src/__test__/votes.test.ts

```diff
@@ -0,0 +1,95 @@
+import { VotingConnectorTheGraph, Vote, Cast } from '../../src'
+
+const VOTING_SUBGRAPH_URL = 'https://api.thegraph.com/subgraphs/name/aragon/aragon-voting-rinkeby'
+const VOTING_APP_ADDRESS = '0xc73e86aab9d232495399d62fc80a36ae52952b81'
+
+describe('when connecting to a voting app', () => {
+  let connector: VotingConnectorTheGraph
+
+  beforeAll(() => {
+    connector = new VotingConnectorTheGraph(
+      VOTING_SUBGRAPH_URL
+    )
+  })
+
+  describe('when querying for all the votes of a voting app', () => {
+    let votes: Vote[]
+
+    beforeAll(async () => {
+      votes = await connector.votesForApp(
+        VOTING_APP_ADDRESS,
+        1000, 0
+      )
+    })
+
+    test('returns a list of votes', () => {
+      expect(votes.length).toBeGreaterThan(0)
+    })
+
+    describe('when looking at a vote', () => {
+      let vote: Vote
+
+      beforeAll(() => {
+        vote = votes[0]
+      })
+
+      test('should be executed', () => {
+        expect(vote.executed).toBe(true)
+      })
+
+      test('should have no nays', () => {
+        expect(vote.nay).toBe('0')
+      })
+
+      // TODO: Unskip when bug in subgraph is fixed
+      test.skip('should have yeas', () => {
```

Comment:
There was a bug in the subgraph which caused this to fail. It's syncing now. Will update by removing this skip once it's done.